.storybook/public/mockServiceWorker.js

```diff
@@ -2,13 +2,14 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (2.1.5).
+ * Mock Service Worker.
  * @see https://github.com/mswjs/msw
  * - Please do NOT modify this file.
  * - Please do NOT serve this file on production.
  */
 
-const INTEGRITY_CHECKSUM = '223d191a56023cd36aa88c802961b911'
+const PACKAGE_VERSION = '2.3.1'
+const INTEGRITY_CHECKSUM = '26357c79639bfa20d64c0efca2a87423'
```

Comment:
what do we reference to change these variable values since I saw `- Please do NOT modify this file.` on top, just trying to understand more about this file.

Reply:
I didnt make those changes, it automatically did that